homeassistant/components/gogogate2/cover.py

```diff
@@ -57,20 +70,20 @@ async def async_setup_entry(
 
     async_add_entities(
         [
-            Gogogate2Cover(config_entry, data_update_coordinator, door)
+            DeviceCover(config_entry, data_update_coordinator, door)
             for door in get_configured_doors(data_update_coordinator.data)
         ]
     )
 
 
-class Gogogate2Cover(CoverEntity):
+class DeviceCover(CoverEntity):
```

Comment:
```suggestion
class DeviceCover(CoordinatorEntity, CoverEntity):
```

We now have a `CoordinatorEntity` for use with `DataUpdateCoordinator`.  
Example conversion: https://github.com/home-assistant/core/pull/39449

It doesn't save you much code in this case though.